x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -183,6 +204,55 @@ export class LoginPage extends Component<Props, State> {
       );
     }
 
+    if (!isCookiesEnabled) {
+      if (isWindowEmbedded()) {
+        return (
+          <EuiEmptyPrompt
+            body={
+              <p>
+                <FormattedMessage
+                  id="xpack.security.loginPage.requiresNewWindowTitle"
+                  defaultMessage="This content can't be viewed embedded in another website."
+                />
+              </p>
+            }
+            actions={
+              <EuiButton
+                href={window.location.href}
+                target="_blank"
+                iconType="popout"
+                iconSide="right"
+                fill
+              >
+                <FormattedMessage
+                  id="xpack.security.loginPage.requiresNewWindowMessage"
+                  defaultMessage="Open in Elastic"
+                />
+              </EuiButton>
+            }
+            style={{ paddingTop: 0 }}
+          />
+        );
+      }
+
+      return (
```

Comment:
It's not that straightforward unfortunately. The Licensing plugin for example uses session storage as well so this would be a bigger effort to ensure all of them fail gracefully and what the behaviour should be.

Reply:
Got it, let's keep it as is then, thanks for investigating! Would you mind filing issues for the Core and App Services teams to handle that case in their code base? I believe Kibana as a whole should deal with this (mis-)configuration in a more user-friendly way.

Reply:
> I believe Kibana as a whole should deal with this (mis-)configuration in a more user-friendly way.

Big ++ from me. We have an existing issue here describing this behavior, but it's currently assigned to us. We should probably close it out in favor of specific issues against `Core` and `AppServices`: https://github.com/elastic/kibana/issues/41444

Reply:
Sounds good, I've opened a separate issue here #121189